Code/GraphMol/new_canon.cpp

```diff
@@ -20,6 +20,78 @@
 
 namespace RDKit {
 namespace Canon {
+
+namespace {
+void flipIfNeeded(Bond::BondStereo &st1,
+                  const canon_atom *const *controllingAtoms) {
```

Comment:
This might be safer as 

```
const canon_atom *const controllingAtoms [4]
```

Reply:
I'd definitely do something like that if this were a function which is callable from elsewhere, but this is only useable from code in this file (anonymous namespace)